Integrations/Flashpoint/Flashpoint.py

```diff
@@ -0,0 +1,1321 @@
+    and call specific function for all values.
+
+    :param client:
+    :param indicator: comma-separated values or single value
+    :param func: reputation command function. i.e file_lookup, domain_lookup etc.
+    :return: output of all value according to specified function.
+    """
+    args = argToList(indicator, ',')
+    for arg in args:
+        return_outputs(*func(client, arg))
+
+
+''' FUNCTIONS '''
+
+
+def ip_lookup_command(client, ip):
+    """
+    'ip' command to lookup a particular ip-address
+
+    :param client:
+    :param ip: ip-address
+    :return: command output
+    """
+    query = r'+type:("ip-src","ip-dst") +value.\*:"' + urllib.parse.quote(ip.encode('utf-8')) + '"'
+    resp = client.http_request("GET", url_suffix=get_url_suffix(query))
+
+    if isinstance(resp, list):
+        indicators = resp
+    else:
+        indicators = []
+
+    if len(indicators) > 0:
+
+        hr = '### Flashpoint IP address reputation for ' + ip + '\n'
+        hr += 'Reputation: Malicious\n\n'
+
+        events_details = get_events_from_ioc_resp(indicators)
+
+        hr += tableToMarkdown('Events in which this IOC observed', events_details['events'],
+                              ['Date Observed (UTC)', 'Name', 'Tags'])
+
+        fp_link = client.url + '/home/search/iocs?group=indicator&ioc_type=ip-dst%2Cip-src&ioc_value=' + ip
+        hr += '\nAll events and details (fp-tools): [{}]({})\n'.format(fp_link, fp_link)
+
+        ec = {outputPaths['ip']: {
+            'Address': ip,
+            'Flashpoint': {
+                'href': events_details['href']
+            }
+        }, 'DBotScore': {
+            'Indicator': ip,
+            'Type': 'ip',
+            'Vendor': 'Flashpoint',
+            'Score': 3
+        }}
+
+        ec[outputPaths['ip']]['Malicious'] = {
+            'Vendor': 'Flashpoint',
+            'Description': 'Found in malicious indicators dataset'
+        }
+
+        return hr, ec, resp
+
+    else:
+
+        torrent_search_url_suffix = '/all/search?query=+basetypes:(+torrent) +is_verified:true ' \
+                                    '+ip_address:("' + urllib.parse.quote(ip.encode('utf-8')) + \
+                                    '")&limit=10&_source_includes=ip_address'
+        torrent_resp = client.http_request("GET", url_suffix=torrent_search_url_suffix)
+        torrent_result = torrent_resp.get('hits').get('hits', [])
+
+        if torrent_result:
+            torrent_search_link = client.url + '/home/search/torrents?ip_address=' + ip
+
+            hr = '### Flashpoint IP address reputation for ' + ip + '\n'
+            hr += 'Reputation: Suspicious\n\n'
+            hr += 'FP tools link to torrent search: [{}]({})\n'.format(torrent_search_link, torrent_search_link)
+
+            ec = {
+                'DBotScore': {
+                    'Indicator': ip,
+                    'Type': 'ip',
+                    'Vendor': 'Flashpoint',
+                    'Score': 2
+                }
```

Comment:
you should add here the `IP` (`outputPaths['ip']`) output too. not only for malicious.
no need to add the malicious part.

Reply:
Added